netci.groovy

```diff
@@ -1357,6 +1369,7 @@ def static addNonPRTriggers(def job, def branch, def isPR, def architecture, def
         case 'jitstressregs0x1000':
         case 'minopts':
         case 'tieredcompilation':
+        case 'no_tiered_compilation':
```

Comment:
Like for `'innerloop'`, should there also be a case for `'no_tiered_compilation_pri0'` that just breaks?

Reply:
👍